lib/ts/recipe/session/api/middleware.ts

```diff
@@ -12,69 +12,35 @@
  * License for the specific language governing permissions and limitations
```

Comment:
This file can be deleted and the functions in this file can be put somewhere else - perhaps in the file where they are being used.

Reply:
functions moved to utils